packages/testing/src/chai-retry-plugin/helpers.ts

```diff
@@ -1,9 +1,22 @@
 import Chai from 'chai';
-import { timeout as timeoutPromise, sleep } from 'promise-assist';
+import { timeout as timeoutPromise, sleep as sleepPromise } from 'promise-assist';
+import { adjustTestTime, mochaCtx } from '../mocha-ctx';
 
 import { chaiMethodsThatHandleFunction } from './constants';
 import type { RetryAndAssertArguments } from './types';
 
+export const addTimeoutSafetyMargin = (ms: number) => mochaCtx() && adjustTestTime(ms);
+
+function sleep(ms: number): Promise<void> {
```

Comment:
maybe `sleepWithSafetyMarginTimeout` or something in that nature?

Reply:
@yurii-ve fixed

Also renamed `timeout` to `timeoutWithSafetyMargin`

Reply:
> Missing PR description

fixed